templates/etc/td-agent/td-agent.conf

```diff
@@ -15,8 +15,8 @@
 <match **>
   type google_cloud
   # Set the chunk limit conservatively to avoid exceeding the limit
-  # of 10MB per write request.
-  buffer_chunk_limit 2M
+  # of 5MB per write request.
```

Comment:
FWIW, I wouldn't change the "10MB" part of the comment since that is the actual API size limit.

I set the buffer_chunk_limit to 2MB to give us reasonable against exceeding the limit - but it's not a great solution because they are measuring different things, it's not 100% reliable, and we're kind of abusing the purpose of this parameter as well.  It basically means that fluentd isn't going to send the plugin more than 2MB of data at one time.  The plugin will try to put all that data into one batch (assuming it's all from the same source, otherwise it might make multiple batches), but even if it is one batch and once we add the per-entry metadata we shouldn't exceed 10MB overall even if the entries are small.  Although, I believe if you send a bunch of 1-byte entries, it will still fail (which I don't think is likely in practice, but...)

So, ideally we would set this parameter to something much higher and let the plugin count bytes and decide for itself how much to put in each batch.